docs/resources/metal_connection.md

```diff
@@ -252,19 +252,64 @@ resource "equinix_fabric_connection" "example" {
+- `type` (String) Connection type - dedicated, shared or shared_port_vlan
+
+### Optional
+
+- `contact_email` (String) The preferred email used for communication and notifications about the Equinix Fabric interconnection
+- `description` (String) Description of the connection resource
+- `facility` (String, Deprecated) Facility where the connection will be created
+- `metro` (String) Metro where the connection will be created
+- `mode` (String) Mode for connections in IBX facilities with the dedicated type - standard or tunnel
+- `organization_id` (String) ID of the organization responsible for the connection. Applicable with type "dedicated"
+- `project_id` (String) ID of the project where the connection is scoped to. Required with type "shared"
+- `service_token_type` (String) Only used with shared connection. Type of service token to use for the connection, a_side or z_side
+- `speed` (String) Connection speed -  Values must be in the format '<number>Mbps' or '<number>Gpbs', for example '100Mbps' or '50Gbps'.  Actual supported values will depend on the connection type and whether the connection uses VLANs or VRF.
+- `tags` (List of String) Tags attached to the connection
+- `vlans` (List of Number) Only used with shared connection. VLANs to attach. Pass one vlan for Primary/Single connection and two vlans for Redundant connection
+- `vrfs` (List of String) Only used with shared connection. VRFs to attach. Pass one VRF for Primary/Single connection and two VRFs for Redundant connection
+
+### Read-Only
+
+- `authorization_code` (String) Fabric Authorization code to configure the NIMF connection with Cloud Service Provider through Equinix Fabric with the [equinix_fabric_connection](./fabric_connection.md) resource from the [Equinix Developer Portal](https://developer.equinix.com/dev-docs/fabric/getting-started/fabric-v4-apis/connect-metal-to-amazon-web-services)
+- `id` (String) The unique identifier of the resource
+- `ports` (List of Object) List of connection ports - primary (`ports[0]`) and secondary (`ports[1]`) (see [below for nested schema](#nestedatt--ports))
+- `service_tokens` (List of Object) Only used with shared connection. List of service tokens required to continue the setup process with [equinix_fabric_connection](https://registry.terraform.io/providers/equinix/equinix/latest/docs/resources/fabric_connection) or from the [Equinix Fabric Portal](https://fabric.equinix.com/dashboard) (see [below for nested schema](#nestedatt--service_tokens))
+- `status` (String) Status of the connection resource
+- `token` (String, Deprecated) Only used with shared connection. Fabric Token required to continue the setup process with [equinix_fabric_connection](https://registry.terraform.io/providers/equinix/equinix/latest/docs/resources/fabric_connection) or from the [Equinix Fabric Portal](https://fabric.equinix.com/dashboard)
+
+<a id="nestedatt--ports"></a>
+### Nested Schema for `ports`
+
+Read-Only:
+
+- `id` (String)
```

Comment:
These have lost their description because it was hard-coded in the docs.  For the `metal_connection` resource you could either keep the hard-coded attributes for now or update the resource and data source schemas under `internal/resources/metal/connection/` to add the missing descriptions for these properties.

Reply:
@ctreatma I have kept hard coded attributes for metal_connections docs.